docs/source/usage.rst

```diff
@@ -5,14 +5,41 @@ Usage
 The driver class is configured to run for the realsense install L515 and D435.
 it may run with other modules but some functionality will give breakdown
 
-Start by importing intel realsense devices and inializing the device
+Please see config_template.conf to see how to structure your config file.
+
+You should be in the intel-realsense-devices directory.
+
+To intialize device:
 
 .. code-block:: python
 
-    from intel_realsense_devices.driver import Device
-    device = Device(config_filename = "YOUR CONFIG YAML FILE", h5py_filename = "YOUR H5PY/HDPY FILE NAME")
-    device.init()
+   import intel_realsense_devices
+   print(f'current version of the library is {intel_realsense_devices.__version__}')
+   from intel_realsense_devices.device import Device
+   device = Device(config_filename = "YOUR CONFIG YAML FILE", h5py_filename = "YOUR H5PY/HDPY FILE NAME")
+   device.init()
+
+Your config yaml file must contain a serial number and imgs_to_collect in order to prepare early. This contatins the number of images 
+you want to collect from the camera. This will be used as the size for the ciruclar buffer. Since the 
+gyroscope and accelerometer collect data at a higher frequency the size of the circular buffer will need be
+accounted for.
```

Comment:
What does it mean that it would be "accounted for"? Dlo you read images acquisition frequency and determine how long gyro and accelerometer need to be?